src/PermissionRegistrar.php

```diff
@@ -59,7 +59,7 @@ public function forgetCachedPermissions()
 
     public function getPermissions(): Collection
     {
-        return $this->cache->remember($this->cacheKey, config('permission.cache_expiration_time'), function () {
+        return $this->cache->forever($this->cacheKey, function () {
```

Comment:
This is a breaking change. Please restore the old behaviour and old versions of readme / config